### DISCUSSION TITLE
Link group

### DISCUSSION THREAD

**Discussion by Guray00** [JulianPrieber/littlelink-custom#151] [Feature requests]
It would be awesome to add links in the same group, like linktree, with a title and maybe a short description.

**Comment by JulianPrieber** [comment]
I think what you are referring to is already possible with spaces and headings.

**Reply by Guray00** [reply]
Ah ok... sorry if I ask but how can I add heading? 😅

**Reply by JulianPrieber** [reply]
When adding a new link/button, if you scroll all the way down in your selection menu, you will find Headings and Spaces.

**Reply by Guray00** [reply]
You are right, thanks you! Sorry for my misunderstanding...
